exercises/concept/meltdown-mitigation/.docs/instructions.md

```diff
@@ -30,13 +30,15 @@ True
 Once the reactor has started producing power its efficiency needs to be determined.
 Efficiency can be grouped into 4 bands:
 
-1. green -> 80-100% efficiency
-2. orange -> 60-79% efficiency
-3. red -> 30-59% efficiency
-4. black -> <30% efficient
-
-These percentage ranges are calculated as `(generated_power/theoretical_max_power)*100`
-where `generated_power = voltage * current`
+1. green -> efficiency of 80% or more,
+2. orange -> efficiency of less than 80% but at least 60%,
```

Comment:
It's generally easier for people to parse ranges from low to high. "efficiency of at least 60% but less than 80%"